scripts/release/build/rpm/sign.sh

```diff
@@ -5,44 +5,44 @@
 set -ex
 
 export HOME=/root
-export GOPATH=${HOME}/go
-export PATH=${GOPATH}/bin:/usr/local/go/bin:${PATH}
+export GOPATH="$HOME/go"
+export PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"
 
-cd ${HOME} && tar jxf /root/subhome/gnupg*.tar.bz2
+cd "$HOME" && tar jxf /root/subhome/gnupg*.tar.bz2
 
-export PATH="${HOME}/gnupg2/bin:${PATH}"
-export LD_LIBRARY_PATH=${HOME}/gnupg2/lib
+export PATH="$HOME/gnupg2/bin:$PATH"
+export LD_LIBRARY_PATH="$HOME/gnupg2/lib"
 
 umask 0077
 mkdir -p ~/.gnupg
 umask 0022
 
-touch "${HOME}/.gnupg/gpg.conf"
-if grep -q no-autostart "${HOME}/.gnupg/gpg.conf"; then
-    echo ""
+touch "$HOME/.gnupg/gpg.conf"
+if grep -q no-autostart "$HOME/.gnupg/gpg.conf"; then
+    echo
 else
-    echo "no-autostart" >> "${HOME}/.gnupg/gpg.conf"
+    echo "no-autostart" >> "$HOME/.gnupg/gpg.conf"
 fi
-rm -f ${HOME}/.gnupg/S.gpg-agent
+rm -f $HOME/.gnupg/S.gpg-agent
 (cd ~/.gnupg && ln -s /root/S.gpg-agent S.gpg-agent)
 
 gpg --import /root/keys/dev.pub
 gpg --import /root/keys/rpm.pub
 rpmkeys --import /root/keys/rpm.pub
-echo "wat" | gpg -u rpm@algorand.com --clearsign
+echo wat | gpg -u rpm@algorand.com --clearsign
```

Comment:
what is this doing?

Reply:
You need to pre-cache the agent so this just executes a throwaway command to do it. "wat" is a Brian special, I think. :)

Reply:
haha nice

Reply:
Yeah, this could be removed, but I've left it in as a kind of tribute :)